src/event/special_user_message.rs

```diff
@@ -358,6 +372,36 @@ fn deploy_command<'a>(
             .send()
             .await
             .map_err(|e| format!("API Error: {:?}", e))?;
+        let res_json = res
+            .json::<BuildsResponse>()
+            .await
+            .map_err(|_| "Invalid API Response")?;
+        banker::message(format!("Build submitted! URL: {}", res_json.setup_log)).await?;
+        Ok(())
+    }
+    .boxed()
+}
+
+lazy_static::lazy_static! {
+    pub static ref RESTART_SERVER: SpecialUserMessageTrigger = SpecialUserMessageTrigger {
+            regex: Regex::new("<@([A-z|0-9]+)> goblin chant").unwrap(),
+            then: &restart_command,
+    };
+}
+
+fn restart_command<'a>(
+    _: regex::Captures<'a>,
+    _: Message,
+    _: &'a Sender<FarmingInputEvent>,
+) -> HandlerOutput<'a> {
+    use std::fs;
+    async move {
+        banker::message(match fs::write("restart", "restarting") {
+            Ok(()) => "Scheduled a restart!".to_string(),
+            Err(e) => e.to_string(),
```

Comment:
just dropping that error in that chat without any explanation probably won't be helpful, it might be better to `format!` it in with some context, probably that the `restart` file couldn't be written to trigger a server restart.